api/nakadi-event-bus-api.yaml

```diff
@@ -797,7 +797,74 @@ paths:
           description: Service (temporarily) unavailable
           schema:
             $ref: '#/definitions/Problem'
+    get:
```

Comment:
I don't think so. We do not have these filters for event type listing endpoint, as well. At some point we will be required to add pagination to all of the listing endpoints.